packages/ui/Logo/index.tsx

```diff
@@ -4,62 +4,52 @@ import Link from "next/link";
-  whiteURL:
+  white:
     "data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjgiIGhlaWdodD0iMzgiIHZpZXdCb3g9IjAgMCAyOCAzOCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZD0iTTAgMTEuNTgwNkMwIDEwLjI2ODQgMC44NTI3NDkgOS4xMDg1MiAyLjEwNTE4IDguNzE3MTNMMjQuMTA1MiAxLjg0MjEzQzI2LjAzNyAxLjIzODQ1IDI4IDIuNjgxNjUgMjggNC43MDU1N1YxOS40MTk0QzI4IDIwLjczMTYgMjcuMTQ3MyAyMS44OTE1IDI1Ljg5NDggMjIuMjgyOUwzLjg5NDgzIDI5LjE1NzlDMS45NjMwNCAyOS43NjE2IDAgMjguMzE4MyAwIDI2LjI5NDRWMTEuNTgwNloiIGZpbGw9IiNGQ0ZDRkMiLz4KPHBhdGggb3BhY2l0eT0iMC42IiBkPSJNMCAxMS43MDU2QzAgOS42ODE2NSAxLjk2MzAzIDguMjM4NDUgMy44OTQ4MiA4Ljg0MjEzTDI1Ljg5NDggMTUuNzE3MUMyNy4xNDczIDE2LjEwODUgMjggMTcuMjY4NCAyOCAxOC41ODA2VjMzLjI5NDRDMjggMzUuMzE4MyAyNi4wMzcgMzYuNzYxNiAyNC4xMDUyIDM2LjE1NzlMMi4xMDUxOCAyOS4yODI5QzAuODUyNzQ5IDI4Ljg5MTUgMCAyNy43MzE2IDAgMjYuNDE5NFYxMS43MDU2WiIgZmlsbD0iI0QyRTFEQiIvPgo8L3N2Zz4K",
 };
 
 type LogoProps = {
-  white?: boolean;
-  logoWidth?: number;
+  $white?: boolean;
+  $logoWidth?: number;
 };
 
-type LogoSignetProps = Pick<LogoProps, "logoWidth" | "white">;
-
-type ContainerProps = {
-  children?: React.ReactNode;
-} & Pick<LogoProps, "logoWidth" | "white">;
-
-const LogoStyledContainer = styled(Link)<ContainerProps>`
-  @import url("https://fonts.googleapis.com/css2?family=Signika:wght@600&display=swap");
+const LogoStyledContainer = styled(Link)<LogoProps>`
   font-family: "Signika", sans-serif;
   font-weight: 600;
   text-decoration: none;
   display: flex;
   justify-content: space-around;
   align-items: center;
-  color: ${({ white }) => (white ? "white" : "#1E4C40")};
-  /*
-	temporarily adding background to white version to make it visible on white background
-	*/
-  background: ${({ white }) =>
-    white ? ({ theme }) => theme.logo.main : "none"};
-  ${({ logoWidth }) =>
-    logoWidth &&
+  color: ${({ $white, theme }) => ($white ? "white" : theme.logo.main)};
+
+  ${({ $logoWidth }) =>
+    $logoWidth &&
     css`
-      width: ${logoWidth + "px"};
-      height: ${(logoWidth * 0.27).toFixed(2) + "px"};
-      font-size: ${(logoWidth / 6).toFixed(2) + "px"};
-      letter-spacing: ${((logoWidth / 6) * -0.02).toFixed(2) + "px"};
+      width: ${$logoWidth + "px"};
+      height: ${($logoWidth * 0.27).toFixed(2) + "px"};
+      font-size: ${($logoWidth / 6).toFixed(2) + "px"};
+      letter-spacing: ${(($logoWidth / 6) * -0.02).toFixed(2) + "px"};
     `}
 `;
 
-const LogoSignet = styled.span<LogoSignetProps>`
-  background: url(${({ white }) => white ? logoVersions.whiteURL : logoVersions.colorURL})
-    no-repeat center / contain;
+const LogoSignet = styled.span<LogoProps>`
+  background-image: url(${({ $white }) => $white && logoVersions.white || logoVersions.color});
```

Comment:
It's kinda hacky condition✍️ . Should also work within the use ternary operator. I think this will be easier to read :D 

```suggestion
  background-image: url(${({ $white }) => $white ? logoVersions.white : logoVersions.color});
```

Reply:
Good point here by @Noshi96! @AzulloCyll could you apply? Afterwards we could merge this to main :).

Reply:
Aplied, but unfortunately we exceeded our deploy limit on vercel. "...try again in 5 hours..." OMG, heavy penalty.